packages/template-package/.eslintrc.cjs

```diff
@@ -0,0 +1,3 @@
+const { createConfig } = require("../../config/v-next/eslint.cjs");
+
+module.exports = createConfig(__filename);
```

Comment:
shouldn't the entry points under `exports` in the `package.json` be passed here as a second arg?

Reply:
done